facade-app/src/test/__snapshots__/Footer.test.js.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Footer renders correctly 1`] = `
+<div>
+  <footer
+    class="rs-footer"
+  >
+    <div
+      class="rs-navbar rs-navbar-default"
+      role="navigation"
+    >
+      <div
+        class="rs-navbar-body"
+        style="text-align: center; height: 30px; font-size: 15px;"
+      >
+        <div>
+          © Copyright 
+          2022
```

Comment:
this will fail next year as date will change.

Reply:
make this as dynamic.

Reply:
good point, changed this so the year can be passed as a prop